tensorboard/manager.py

```diff
@@ -409,8 +406,8 @@ def start(arguments, timeout=datetime.timedelta(seconds=60)):
     os.close(stderr_fd)
 
   poll_interval_seconds = 0.5
-  end_time = start_time + timeout
-  while datetime.datetime.now() < end_time:
+  end_time = start_time + timeout.total_seconds()
```

Comment:
optional: any thoughts on putting suffix of "_s" on `end_time` to denote the unit?

Reply:
Sure. The unit is really “seconds since epoch”, not “seconds”, but
adding a `_seconds` suffix to these local variables probably can’t hurt.